arches/app/views/user.py

```diff
@@ -159,12 +164,26 @@ def post(self, request):
                 user = form.save()
                 try:
                     admin_info = settings.ADMINS[0][1] if settings.ADMINS else None
+                    email_username = user.username
+                    # Use firstname for email if we have it.
+                    if user.first_name:
+                        email_username = user.first_name
+
                     message = (
                         f"Your {settings.APP_NAME} profile was just changed.  If this was unexpected, please contact your "
                         f"{settings.APP_NAME} administrator{f' at {admin_info}.' if (admin_info and not str.isspace(admin_info)) else '.'}"
                     )
                     message = _(message)
-                    user.email_user(_("Your " + settings.APP_NAME + " Profile Has Changed"), message)
+
+                    email_context = return_message_context(message,"",None,{"username":email_username})
+
+                    html_content = render_to_string("email/general_notification.htm", email_context)  # ...
+                    text_content = strip_tags(html_content)  # this strips the html, so people will have the text as well.
+
+                    # create the email, and attach the HTML version as well.
+                    msg = EmailMultiAlternatives(_("Your " + settings.APP_NAME + " Profile Has Changed!"), text_content, admin_info, [form.cleaned_data["email"]])
```

Comment:
I'm unsure of how well the i18n software will handle `_("Your " + settings.APP_NAME + " Profile Has Changed!")`

Could this be changed to `_("Your {settings.APP_NAME} Profile Has Changed!")`?

Reply:
@chrabyrd Changed as requested